Specs/customizeJasmine.js

```diff
@@ -26,6 +28,12 @@ function customizeJasmine(
       return;
     }
 
+    if (includedName && name !== includedName) {
```

Comment:
Could we include partial matches? That way we can include all tests in a category. For example @ptrgags said he uses the web browser to run all `ModelExperimental` tests, which span multiple files.

Reply:
So just for some more information on this, the SpecRunner query seems to do a case-sensitive substring search.

Though I think even a regex search would be easy to add with `new RegExp(query)` -- might be nice? then you could run custom queries like `--includeName 'Cesium3DTileset|ModelExperimental'`. Not necessary but might be nice to have